leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -185,6 +185,8 @@ var Config = Backbone.View.extend({
             }
         }
         this.model.destroy();
+        IntentionColoring.setColorMode("EVO", undefined);
```

Comment:
Is IntentionColoring still being used? I thought we were removing it from evoObjects